src/components/shared/filterDropdownButton/index.js

```diff
@@ -10,13 +10,15 @@ import IntegerFilter from './integerFilter';
 import TextFilter from './textFilter';
 import styles from './filterContainer.css';
 import AddressFilter from './addressFilter';
+import DropdownFilter from './dropdownFilter';
 
 const filterComponents = {
   'date-range': DateFieldGroup,
   'number-range': AmountFieldGroup,
   text: TextFilter,
   integer: IntegerFilter,
   address: AddressFilter,
+  dropdown: DropdownFilter,
```

Comment:
💅 Given `DropdownFilter` uses `Select` component and not `Dropdown` component, I suggest renaming for consistency.
```suggestion
  select: SelectFilter,
```

Reply:
✅